src/easier68k/core/opcodes/move.py

```diff
@@ -0,0 +1,216 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.enum.op_size import MoveSize
+from ...core.enum.ea_mode_bin import EAModeBinary
+from ...simulator.m68k import M68K
+from ...core.opcodes.opcode import Opcode
+
+
+class Move(Opcode):
+    pass
+
+
+class Move(Opcode):
```

Comment:
You define move twice?

Reply:
Oh, I forgot to remove that, it was necessary before and then Chris and I decided on a restructure that made it not necessary: I'll remove it